invenio_stats/config.py

```diff
@@ -35,11 +37,23 @@
 )
 """Default exchange for message queue."""
 
-STATS_INDICIES_PREFIX = 'events'
+STATS_INDICES_PREFIX = 'events'
 """Allowed event types."""
 
 STATS_REGISTER_RECEIVERS = True
 """Register signal receivers."""
 
 STATS_INDICES_SUFFIX = '%Y.%W',
 """Suffix of indices."""
+
+CELERY_BEAT_SCHEDULE = {
+    'indexer': {
+        'task': 'invenio_stats.tasks.index_events',
+        'schedule': timedelta(seconds=5),
```

Comment:
Could you replace the 5 seconds with 5 minutes? Otherwise we might have issues as Lars mentioned in some other PR.

Note: This means that we should not make the aggregations at midnight or we might miss some events, they should be done a few hours later just to be sure.